README.md

```diff
@@ -63,9 +61,10 @@ Your Answer:
 argument. Finally, demonstrate calling `doSomething` with a function.**
 
 Your Answer:
-```js
-// write code here
-```
+var doSomething = function(thingToDo){
+  return thingToDo;
+}
+
 
```

Comment:
Close. You actually want to call the function passed in as an argument (`thingToDo`), not return it. Otherwise, the callback will never be run.

And don't forget to call `doSomething` after the function definition (e.g., `doSomething( sampleFunction )`).